src/main/java/com/synopsys/integration/detect/lifecycle/run/operation/OperationFactory.java

```diff
@@ -283,7 +283,7 @@ public final File generateRapidJsonFile(NameVersion projectNameVersion, List<Dev
     }
 
     public final void publishRapidResults(File jsonFile, RapidScanResultSummary summary) throws DetectUserFriendlyException {
-        auditLog.namedInternal("Publish Rapid Results", () -> statusEventPublisher.publishDetectResult(new RapidScanDetectResult(jsonFile.getCanonicalPath(), summary)));
+        auditLog.namedInternal("Publish Rapid Results", "Rapid", () -> statusEventPublisher.publishDetectResult(new RapidScanDetectResult(jsonFile.getCanonicalPath(), summary)));
```

Comment:
This is an example of how to phone home an operation. Rapid scans aren't their own tool, so it doesn't get automatically reported.